network-api/networkapi/mozfest/blocks/dark_quote_block.py

```diff
@@ -0,0 +1,17 @@
+from wagtail import blocks
+
+
+class DarkSingleQuoteBlock(blocks.StructBlock):
+    # This is a copy of the SingleQuoteBlock from
+    # wagtailpages.pagemodels.customblocks.single_quote_block.py
+    # There is obviously code duplication here, but it's necessary due to the
+    # styling changes and the different template.
+    # And explicit is better than implicit.
+    quote = blocks.RichTextBlock(features=["bold"])
+    attribution = blocks.CharBlock(required=False)
+    attribution_info = blocks.RichTextBlock(required=False, features=["bold", "link", "large"])
+
+    class Meta:
+        template = "fragments/blocks/dark_quote_block.html"
```

Comment:
That will map to the mozfest directory.

Ideally, I think this block and it's template should stay in the mozfest directory because the other sites templates haven't been prepared for a dark quote block.